src/queue/execQueue.js

```diff
@@ -0,0 +1,103 @@
+// Setup for exec queue which pushes jobs to Redis
+
+const Queue = require('bee-queue');
+const logger = require('../server/logger.js');
+const MeasureReportBuilder = require('fqm-execution').MeasureReportBuilder;
+
+// Create a new queue to establish new Redis connection
+const execQueue = new Queue('exec', {
+  redis: {
+    host: process.env.REDIS_HOST || 'localhost',
+    port: process.env.REDIS_PORT || 6379
+  },
+  removeOnSuccess: true,
+  isWorker: false
+});
+
+execQueue.on('error', err => {
+  logger.error('exec-queue error: ', err);
+});
+
+class ScaledCalculation {
+  constructor(measureBundle, patientIds, periodStart, periodEnd) {
+    this._measureBundle = measureBundle;
+    this._periodStart = periodStart;
+    this._periodEnd = periodEnd;
+
+    // See if count/worker count is less than max job size. if it is use that as jobSize other wise use max job size
+    let jobSize = Math.ceil(patientIds.length / parseInt(process.env.EXEC_WORKERS));
+    if (jobSize > parseInt(process.env.SCALED_EXEC_MAX_JOBSIZE)) {
+      jobSize = parseInt(process.env.SCALED_EXEC_MAX_JOBSIZE);
+    }
+
+    // Prepare measure report builder
+    try {
+      this._mrBuilder = new MeasureReportBuilder(this._measureBundle, {
+        measurementPeriodStart: this._periodStart,
+        measurementPeriodEnd: this._periodEnd,
+        reportType: 'summary'
+      });
+      logger.info(`exec-queue: Created MRBuilder for ${this._mrBuilder.measure.id}`);
+    } catch (e) {
+      logger.error(e);
+      throw new Error('Could not prepare report builder. Measure bundle may not have measure.', e);
```

Comment:
I noticed that `e` containing the actual error message doesn't get bubbled up back to the operation outcome. Instead of speculating that the error may be because it doesn't have a measure, could we just include `e.message` in the re-throw?

E.g.

``` Javascript
throw new Error(`Could not prepare report builder: ${e.message}`);
```

Might be worth testing what happens with the above statement in cases where the measure bundle is corrupt, just to make sure it's sane.

Reply:
Definitely plan to add testing for this. Will look into making sure that comes through.

Reply:
Made it happen in ce74e97.